infracost/1.0.0/README.md

```diff
@@ -0,0 +1,64 @@
+# Infracost Integration with Keptn
+
+![infracost running in keptn](./assets/infracost-keptn.png)

```

Comment:
Relative paths are currently not supported (see [here](https://github.com/keptn-contrib/artifacthub#adding-images-to-the-readme-or-artifacthub-pkgyml-file))